lib/friendly_uuid.rb

```diff
@@ -49,10 +48,7 @@ def compact(uuid)
     end
 
     def expand_to_record(short_uuid)
-      raise ActiveRecord::RecordNotFound unless short_uuid
-      record = self.possible_expansions(short_uuid).first
-      raise ActiveRecord::RecordNotFound unless record
-      record
+      self.possible_expansions(short_uuid).first
```

Comment:
We should no longer need to implicitly raise `ActiveRecord::RecordNotFound` errors since that will be handled by the superclass implementation of `.find` (i.e., `ActiveRecord`'s implementation of it).